src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
```

Comment:
The context meaning of this sentence is that user typed two password, passwords are not same with each other. Is there any better solution for this sentence?